packages/clerk-js/src/ui/components/CreateOrganization/CreateOrganizationPage.tsx

```diff
@@ -1,175 +1,27 @@
-  };
-
-  const onChangeName = (event: React.ChangeEvent<HTMLInputElement>) => {
-    nameField.setValue(event.target.value);
-    updateSlugField(createSlug(event.target.value));
-  };
-
-  const onChangeSlug = (event: React.ChangeEvent<HTMLInputElement>) => {
-    updateSlugField(event.target.value);
-  };
-
-  const updateSlugField = (val: string) => {
-    slugField.setValue(val);
-  };
+  const { mode, navigateAfterCreateOrganization, skipInvitationScreen } = useCreateOrganizationContext();
 
   return (
-    <Wizard {...wizard.props}>
-      <ContentPage
-        Breadcrumbs={null}
-        headerTitle={title}
-        sx={t => ({ minHeight: t.sizes.$60 })}
-      >
-        <Form.Root onSubmit={onSubmit}>
-          <OrganizationProfileAvatarUploader
-            organization={{ name: nameField.value }}
-            onAvatarChange={async file => await setFile(file)}
-            onAvatarRemove={file ? onAvatarRemove : null}
-          />
-          <Form.ControlRow elementId={nameField.id}>
-            <Form.Control
-              sx={{ flexBasis: '80%' }}
-              autoFocus
-              {...nameField.props}
-              onChange={onChangeName}
-              required
-            />
-          </Form.ControlRow>
-          <Form.ControlRow elementId={slugField.id}>
-            <Form.Control
-              sx={{ flexBasis: '80%' }}
-              {...slugField.props}
-              onChange={onChangeSlug}
-              icon={QuestionMark}
-              required
-            />
-          </Form.ControlRow>
-          <FormButtonContainer>
-            <Form.SubmitButton
-              block={false}
-              isDisabled={!canSubmit}
-              localizationKey={localizationKeys('createOrganization.formButtonSubmit')}
-            />
-            {mode === 'modal' && (
-              <Form.ResetButton
-                localizationKey={localizationKeys('userProfile.formButtonReset')}
-                block={false}
-                onClick={closeCreateOrganization}
-              />
-            )}
-          </FormButtonContainer>
-        </Form.Root>
-      </ContentPage>
-      <ContentPage
-        Breadcrumbs={null}
-        headerTitle={inviteTitle}
-        sx={t => ({ minHeight: t.sizes.$60 })}
-      >
-        {organization && (
-          <InviteMembersForm
-            organization={organization}
-            resetButtonLabel={localizationKeys('createOrganization.invitePage.formButtonReset')}
-            onSuccess={wizard.nextStep}
-            onReset={completeFlow}
-          />
-        )}
-      </ContentPage>
-      <SuccessPage
-        title={title}
-        contents={<InvitationsSentMessage />}
-        sx={t => ({ minHeight: t.sizes.$60 })}
-        onFinish={completeFlow}
-      />
-    </Wizard>
+    <CreateOrganizationForm
+      skipInvitationScreen={skipInvitationScreen}
+      navigateAfterCreateOrganization={navigateAfterCreateOrganization}
+      flow={'organizationList'}
```

Comment:
Should this be `default`?

```suggestion
      flow={'default'}
```

Reply:
@BRKalow good catch